### ISSUE TITLE
Gauge: Centering of gauge

### ISSUE THREAD

**Issue by maemigh** [grafana/grafana#31470]
**What would you like to be added**:
Very small detail, but gauge appears to be left aligned rather than center (for a single gauge).

**Why is this needed**:
It would be slightly more visually appealing, especially as compared to the centering of the title.
<img width="117" alt="Screen Shot 2021-02-25 at 19 59 25" src="https://user-images.githubusercontent.com/4478053/109143992-fe044780-77a3-11eb-9dc7-8211eefb932c.png">

**Comment by maemigh** [comment]
Should have made a better screenshot using a panel that has a background:
<img width="131" alt="Screen Shot 2021-02-25 at 20 08 46" src="https://user-images.githubusercontent.com/4478053/109145073-52f48d80-77a5-11eb-9b65-345c45ba5a82.png">

**Comment by torkelo** [comment]
![Screenshot from 2021-02-25 14-01-32](https://user-images.githubusercontent.com/10999/109157224-09d71600-7772-11eb-8ef2-2fadbacf64ff.png)

unable to replicate, can you share panel json? Are you using the gauge panel or old singlestat with gauge enabled?

**Comment by maemigh** [comment]
@torkelo There is some sort of css fault on my end, where the css is coming from another panel. I copied the gauge panel into a fresh dashboard, but the problem still existed until a "shift-reload" on the browser. So this issue can probably be closed, but I'm curious about the css because I thought they were isolated per panel?

**Comment by maemigh** [comment]
To provide more info, this comes from the css when using my hacked version of https://github.com/briangann/grafana-datatable-panel. I only noticed after testing from a slow, tethered connection and things changed after the panel loaded. I did change some css from the original panel, but pretty sure it was all in the datatables wrapper. 

I can test more later if it was a change of mine, but I suspect this panel existing on the dashboard throws other panels off (assuming it loads last).
